test/lmo.jl

```diff
@@ -532,3 +532,43 @@ end
     vvec = FrankWolfe.compute_extreme_point(lmo, [dinf; d1]; direction_indices=(1:10, 11:15))
     @test vvec ≈ [vinf; v1]
 end
+
+
```

Comment:
create a sub-testset for each of your individual tests and give them a name with what you are testing in all cases